src/utils/defineTransform.js

```diff
@@ -1,6 +1,8 @@
 import { createClass, createMethod } from './classUtils';
 import replaceRefs from './replaceRefs';
 
+const transformInlineMap = (path) => path.value.arguments.length === 2 ?  path.value.arguments[0].value : 'Model';
```

Comment:
Why call this Model? Can it just be a class expression without a name?

Reply:
Done!